playbooks/robusta_playbooks/common_actions.py

```diff
@@ -31,3 +31,49 @@ def customise_finding(event: ExecutionBaseEvent, params: FindingOverrides):
         FindingSeverity[params.severity] if params.severity else None
     )
     event.override_finding_attributes(params.title, params.description, severity)
+
+
+class FindingFields(ActionParams):
+    """
+    :var title: Finding title. Title can be templated with name/namespace/kind/node of the resource, if applicable
+    :var aggregation_key: Identifier of this finding
+    :var description: Finding description. Description can be templated
+    :var severity: Finding severity. Allowed values: DEBUG, INFO, LOW, MEDIUM, HIGH
+
+    :example title: "Job $name on namespace $namespace failed"
+    :example aggregation_key: "Job Failure"
+    :example severity: DEBUG
+    """
+
+    title: str
+    aggregation_key: str
+    description: Optional[str] = None
+    severity: Optional[str] = "HIGH"
+
+
+@action
+def create_finding(event: ExecutionBaseEvent, params: FindingFields):
+    """
+    Create a new finding.
+
+    All messages from Robusta are represented as a Finding object.
+
+    This action lets create a Finding that Robusta sends.
```

Comment:
"This action lets create a Finding that Robusta sends." English errors

Reply:
fixed